### PR TITLE
A11y fix ada Information Line background

### PR DESCRIPTION
**Pull request by JackRyan1989**
Addresses item 1 of this ticket: https://github.com/usdoj-crt/beta.ADA.gov-Project-Management/issues/358

Rather than wrapping the Info line text in a div element, I wrapped in a p element and incorporated a <br/> to separate the two lines. As far as my testing showed, it behaves identically as the live site currently does in terms of screen reader functionality and keyboard control.

This allows axe-dev tools to determine that the background color is of sufficient contrast with the text (13.75, AAA).

### CODE REVIEW

**Comment by JackRyan1989** [comment]
@c2nelson We can discuss what this fixes in our next meeting.

**Comment by JackRyan1989** [comment]
@c2nelson I checked with Terri Youngblood today (1/13) regarding the fix I put in place. She has signed off on implementing it.

File: _includes/phone.html [hunk 1]
```diff
@@ -1,14 +1,28 @@
 {% assign lang = site.active_lang %}
 
 <div
-  class="crt-phone display-none desktop:display-flex flex-align-center bg-primary-darker text__reverse radius-bottom-md position-absolute pin-right pin-top margin-right-4 padding-x-3 padding-y-2"
+  class="
```
Comment (billyhunt) [inline _includes/phone.html:4]: Is there a line below class and before all the classes?

Comment (billyhunt) [reply _includes/phone.html:4]: Maybe format this a little nicer?  Whatever you think is reasonable.

File: _includes/phone.html [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
 >
   {% asset ic_phone-circle.svg class="margin-right-2" alt="phone" %}
   <div>
-    <div>
-      <a href="{{ site.contact.link | relative_url }}" class="crt-phone">{{ site.contact.link_title }}</a>
-    </div>
+    <p class="margin-y-0">
+    <a href="{{ site.contact.link | relative_url }}" class="crt-phone"
+      >{{ site.contact.link_title }}</a
+    > <br/>
```
Comment (billyhunt) [inline _includes/phone.html:23]: Are <br />'s still cool?  Could you use css to space it correctly?

File: _includes/phone.html [hunk 3]
```diff
@@ -1,14 +1,28 @@
...
+    padding-x-3 padding-y-2
+  "
 >
   {% asset ic_phone-circle.svg class="margin-right-2" alt="phone" %}
   <div>
-    <div>
-      <a href="{{ site.contact.link | relative_url }}" class="crt-phone">{{ site.contact.link_title }}</a>
-    </div>
+    <p class="margin-y-0">
+    <a href="{{ site.contact.link | relative_url }}" class="crt-phone"
```
Comment (billyhunt) [inline _includes/phone.html:21]: Looks like it should be indented.  Maybe lets try and get a formatter working here.  Thoughts?

**Review by billyhunt — APPROVED** [review]
I put some formatting suggestions in here.  Otherwise it looks good!

File: _includes/phone.html (hunk 1 above)
Comment (JackRyan1989) [reply _includes/phone.html:4]: I'll go with Prettier.

File: _includes/phone.html (hunk 2 above)
Comment (JackRyan1989) [reply _includes/phone.html:23]: I'm not sure what you mean by " 's " ?

File: _includes/phone.html (hunk 3 above)
Comment (JackRyan1989) [reply _includes/phone.html:21]: Yep totally agree. Gonna get Prettier to run automatically on file save. Could be a good idea to have it run automatically on push?

File: _includes/phone.html (hunk 2 above)
Comment (billyhunt) [reply _includes/phone.html:23]:
```
<br />
```
is what I wrote, I guess it interpreted the HTML. LOLZ

File: _includes/phone.html (hunk 3 above)
Comment (billyhunt) [reply _includes/phone.html:21]: Probably.  I run on save from my IDE personally.  If you do it on push get it set up on Cindy's machine too.

**Pull request merged**